### ISSUE TITLE
Use variable names as-is, in Tailwind code.

### ISSUE THREAD

**Issue by davestewart** [bernaferrari/FigmaToCode#107]
Hey,

Great plugin! Clearly lots of work too, thank you.

It seems we're having similar issues to #102 as our Tailwind config is also customised. I explained it more in the [Figma Discord](https://discord.com/channels/675194100147945497/1019657911246520342/threads/1255834578749231126) thinking it might be a general probem.

Even though adding TW config would be useful, we are attempting to move to variables in order to build a design system. 

As such, (and having looked the [code](https://github.com/bernaferrari/FigmaToCode/blob/7066b2745c87c620e298b779ecae3e6d94896f1c/packages/backend/src/tailwind/builderImpl/tailwindColor.ts)) it seems that rather than picking the nearest color, when using a color variable, the plugin could convert the variable name to a Tailwind token name directly, i.e.:

```
button/outlined -> {property}-button-outlined (not {property}-stone-400)
```

Does that make sense?

**Comment by davestewart** [comment]
So I've made a bit of progress identifying variables and returning them to the UI:

![CleanShot 2024-06-28 at 12 52 02](https://github.com/bernaferrari/FigmaToCode/assets/132681/5ae664c9-f427-406d-9dec-b62ff4602947)

The problem arises converting the variables to color names, as the function is async, but the plugin's code is all sync.

```js
var variableId = figma.currentPage.selection[0].fills[0].boundVariables.color.id
var fill = await figma.variables.getVariableByIdAsync(variableId)
console.log(fill.name) // toggle/green
```

At this early stage, rather than modifying all methods and functions impacting the API, I've opted to return the variable ID; perhaps just the rendering functions could do the async calls?

Anyway. Tinkering away here...
